src/styles/main.scss

```diff
@@ -1,3 +1,11 @@
-body {
-  margin: 0;
+@import "./reset";
+@import "./utils/_variables";
+@import "./blocks/calendar";
+
+.main {
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
 }
```

Comment:
If you have `main` block you need a separate file `main.scss` in blocks folder for this part.